readthedocs_ext/readthedocs.py

```diff
@@ -50,10 +50,11 @@ def finalize_media(app):
     # Pull project data from conf.py if it exists
     context = app.builder.config.html_context
     STATIC_URL = context.get('STATIC_URL', DEFAULT_STATIC_URL)
-
-    app.builder.script_files.append(
-        '%sjavascript/readthedocs-doc-embed.js' % STATIC_URL
-    )
+    js_file = '{}javascript/readthedocs-doc-embed.js'.format(STATIC_URL)
+    if sphinx.version_info < (1, 8):
+        app.add_javascript(js_file)
```

Comment:
I'd like to probably keep the logic the same as the existing logic, since there could be subtle changes in how this works across Sphinx versions.